x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysColumnsTests.java

```diff
@@ -287,7 +287,8 @@ private int executeCommandInOdbcModeAndCountRows(String sql) {
             false,
             false,
             null,
-            null
+            null,
+            false
```

Comment:
Maybe these can be randomized (not that it makes much sense to pass `true` for these commands, just to validate that the input is accepted correctly)

Reply:
I guess these values were inherited from `SqlTestUtils.TEST_CFG`; some of these could be indeed randomised, but to stay consistent more changes would be needed in this class. I would limit this PR's scope.